src/components/SongList/SongListItem.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from "react";
+import { GetAllMusic_musicNFTs } from "../../graph-ql/queries/GET_ALL_MUSIC/__generated__/GetAllMusic";
+import { fetchIpfs, ipfsToHttps } from "../../services/ipfs/fetchIpfs";
+import { MusicNftMetaData } from "../../types/MusicNFTData";
+import styled from "styled-components";
+import Image from "next/image";
+type Props = {
+  musicNft: GetAllMusic_musicNFTs;
+  onPlaySong: () => void;
+};
+
+const SongListItemStyled = styled.div`
```

Comment:
Won't Ant design library and tailwiind be enough to get the layout you went for?

Reply:
Complicated such as changing nested div depending on hover on parent div

Reply:
No problem. We can work on that later. Not really an issue now. Lets just focus on shipping!